core/src/components/alert/test/basic/alert.e2e.ts

```diff
@@ -0,0 +1,176 @@
+      const ionLoadingDidDismiss = await page.spyOnEvent('ionLoadingDidDismiss');
+
+      const alert = page.locator('ion-alert');
+
+      await page.click('#asyncHandler');
+      await ionAlertDidPresent.next();
+
+      await page.click('.alert-button');
+
+      await expect(alert).toBeVisible();
+
+      await ionLoadingDidDismiss.next();
+      await ionAlertDidDismiss.next();
+
+      await expect(alert).toBeHidden();
+    });
+  });
+});
+
+test.describe('should not have visual regressions', () => {
+  configs().forEach(({ config, screenshot, title }) => {
+    test(title('header, subheader, message'), async ({ page }) => {
+      await page.goto('/src/components/alert/test/basic', config);
+
+      const alertFixture = new AlertFixture(page, screenshot);
+
+      await alertFixture.open('#basic');
+      await alertFixture.screenshot('basic');
+    });
+
+    test(title('long message'), async ({ page }) => {
+      await page.goto('/src/components/alert/test/basic', config);
+
+      const alertFixture = new AlertFixture(page, screenshot);
+
+      await alertFixture.open('#longMessage');
+      await alertFixture.screenshot('longMessage');
+    });
+
+    test(title('more than two buttons'), async ({ page }) => {
+      await page.goto('/src/components/alert/test/basic', config);
+
+      const alertFixture = new AlertFixture(page, screenshot);
+
+      await alertFixture.open('#multipleButtons');
+      await alertFixture.screenshot('multipleButtons');
+    });
+
+    test(title('no message'), async ({ page }) => {
+      await page.goto('/src/components/alert/test/basic', config);
+
+      const alertFixture = new AlertFixture(page, screenshot);
+
+      await alertFixture.open('#noMessage');
+      await alertFixture.screenshot('noMessage');
+    });
+
+    test(title('two buttons'), async ({ page }) => {
+      await page.goto('/src/components/alert/test/basic', config);
+
+      const alertFixture = new AlertFixture(page, screenshot);
+
+      await alertFixture.open('#confirm');
+      await alertFixture.screenshot('confirm');
+    });
+
+    test(title('form prompt'), async ({ page }) => {
+      await page.goto('/src/components/alert/test/basic', config);
+
+      const alertFixture = new AlertFixture(page, screenshot);
+
+      await alertFixture.open('#prompt');
+      await alertFixture.screenshot('prompt');
+    });
+
+    test(title('radios'), async ({ page }) => {
+      await page.goto('/src/components/alert/test/basic', config);
+
+      const alertFixture = new AlertFixture(page, screenshot);
+
+      await alertFixture.open('#radio');
+      await alertFixture.screenshot('radio');
+    });
+
+    test(title('checkboxes'), async ({ page }) => {
+      await page.goto('/src/components/alert/test/basic', config);
+
+      const alertFixture = new AlertFixture(page, screenshot);
+
+      await alertFixture.open('#checkbox');
+      await alertFixture.screenshot('checkbox');
+    });
+  });
+});
+
+class AlertFixture {
```

Comment:
Many of the other tests share this functionality. Would it make sense to pull the fixture into a separate file and import it?